subprojects/android-test/test-annotations/build.gradle.kts

```diff
@@ -3,3 +3,7 @@ plugins {
     `maven-publish`
     id("com.jfrog.bintray")
 }
+
+dependencies {
+    implementation(project(":subprojects:common:report-viewer"))
```

Comment:
It's not clear why do we need report-viewer in test-annotations module.
Maybe we have problems with module naming?

Reply:
yep, `report-viewer` -> `report` and annotations should be there